toast.yml

```diff
@@ -0,0 +1,30 @@
+image: golang:1.12-alpine
+tasks:
+  apk_update:
+    command: apk update
+  install_protoc_deps:
+    dependencies:
+      - apk_update
+    command: apk add protobuf protobuf-dev git
+  install_protoc_gen_go:
+    dependencies:
+      - install_protoc_deps
+    command: go get -u github.com/golang/protobuf/protoc-gen-go github.com/golang/protobuf/jsonpb github.com/golang/protobuf/proto github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway google.golang.org/grpc
```

Comment:
i'm a little bit nervous about these not being vendored properly, and that causing a lot of problems. protocc was meant to solve these problems